overwatch/processing/detectors/TPC.py

```diff
@@ -397,5 +424,57 @@ def projectToXZ(subsystem, hist, processingOptions, aSide):
     # NOTE: The ``histName`` of the ``histogramContainer`` corresponds to the name of the histogram (container) that we
     #       created. Thus, we can use it here to set the projection to the proper histogram name.
     tempHist.SetName(hist.histName)
+    tempHist.SetTitle(hist.histName)
+    if "DCAr" in hist.histName:
+        tempHist.GetYaxis().SetTitle("DCAr (cm)")
+    if "DCAz" in hist.histName:
+        tempHist.GetYaxis().SetTitle("DCAz (cm)")
+
+    return tempHist
+
+def projectToYZ(subsystem, hist, processingOptions):
+    """ Project a given TH3 histogram onto the YZ axis.
+        
+    This function was built to get an eta vs. phi histograms for positive and negative tracks.
+    
+    Args:
+    subsystem (subsystemContainer): Subsystem which contains the projected histogram.
+    hist (histogramContainer): Histogram container corresponding to the projected histogram.
+    When this function is called, it contains the histogram to project from, so the hist
+    to project from can be retrieved via ``hist.hist``.
+    processingOptions (dict): Dictionary of processing options for the given subsystem.
+    
+    Returns:
+    ROOT.TH2: The projected histogram
+    """
+
+    tempHist = hist.hist.Project3D("YZ")
+    tempHist.SetName(hist.histName)
+    tempHist.SetTitle(hist.histName)
 
     return tempHist
+
+def projectTo1D(subsystem, hist, processingOptions):
+    """ Project a given TH3 histogram onto the Z or Y axis.
+        
+        This function was built to get P_T or eta histograms for positive and negative tracks.
+        
+        Args:
+        subsystem (subsystemContainer): Subsystem which contains the projected histogram.
+        hist (histogramContainer): Histogram container corresponding to the projected histogram.
+        When this function is called, it contains the histogram to project from, so the hist
+        to project from can be retrieved via ``hist.hist``.
+        processingOptions (dict): Dictionary of processing options for the given subsystem.
+        
+        Returns:
+        ROOT.TH1: The projected histogram
+        """
```

Comment:
You have kind of the opposite indentation issue here. The second line is indented 4 too far, as is the "Args" header, the "Returns" header, and the last `"""`.